app/views/layouts/application.html.erb

```diff
@@ -8,6 +8,8 @@
   <%= csrf_meta_tags %>
 </head>
 <body>
+  <%= notice %>
```

Comment:
Consider bringing in the flash rendering code from Bloccit as it'll show the flash[:error] messages too.